src/pages/Thanks/ThanksPage.vue

```diff
@@ -1,91 +1,99 @@
 <template>
 	<www-page data-testid="thanks-page">
-		<div class="row page-content" v-if="receipt && !showFocusedShareAsk">
-			<div class="small-12 columns thanks">
-				<div class="thanks__header hide-for-print">
-					<template v-if="receipt">
-						<h1
-							class="tw-mt-1 tw-mb-3"
-						>
-							Thank you!
-						</h1>
-						<p
-							v-if="loans.length > 0"
-							class="thanks__header-subhead tw-text-subhead tw-mb-2"
-							data-testid="thanks-message"
-						>
-							Thanks for supporting
-							<span class="data-hj-suppress">{{ borrowerSupport }}</span>.<br>
-						</p>
-						<p v-if="lender.email" class="hide-for-print">
-							We've emailed your order confirmation to
-							<strong class="data-hj-suppress ">{{ lender.email }}</strong>
-						</p>
-						<p v-else class="hide-for-print">
-							We've emailed your order confirmation to you.
-						</p>
-					</template>
+		<template v-if="isOnlyDonation">
```

Comment:
Are we running this outside an experiment?

Reply:
Yes, we've already run this as an experiment in Optimizely. This ticket is committing the changes from that experiment to production.